### PR TITLE
Fix CLI deployment cancellation

### PR DESCRIPTION
**Pull request by ironcladlou**
Fix a regression which caused deployer pods from being selected for
cancellation.

### CODE REVIEW

**Comment by ironcladlou** [comment]
cc @abhgupta

File: pkg/cmd/cli/cmd/deploy.go [hunk 1]
```diff
@@ -300,6 +302,10 @@ func (c *cancelDeploymentCommand) cancel(config *deployapi.DeploymentConfig, out
 	if err != nil {
 		return err
 	}
+	if len(deployments.Items) == 0 {
+		fmt.Println("no deployments found to cancel")
```
Comment (smarterclayton) [inline pkg/cmd/cli/cmd/deploy.go]: this should be printing to out

File: pkg/cmd/cli/cmd/deploy.go [hunk 2]
```diff
@@ -319,9 +325,11 @@ func (c *cancelDeploymentCommand) cancel(config *deployapi.DeploymentConfig, out
 			if err == nil {
 				fmt.Fprintf(out, "cancelled #%d\n", config.LatestVersion)
 			} else {
-				fmt.Fprintf(out, "couldn't cancel deployment %d (status: %s): %v", deployutil.DeploymentVersionFor(&deployment), status, err)
+				fmt.Fprintf(out, "couldn't cancel deployment %d (status: %s): %v\n", deployutil.DeploymentVersionFor(&deployment), status, err)
 				failedCancellations = append(failedCancellations, strconv.Itoa(deployutil.DeploymentVersionFor(&deployment)))
 			}
+		default:
+			fmt.Println("no active deployments to cancel")
```
Comment (smarterclayton) [inline pkg/cmd/cli/cmd/deploy.go]: out

File: pkg/cmd/cli/cmd/deploy.go (hunk 1 above)
Comment (ironcladlou) [inline pkg/cmd/cli/cmd/deploy.go]: Fixed

File: pkg/cmd/cli/cmd/deploy.go (hunk 2 above)
Comment (ironcladlou) [inline pkg/cmd/cli/cmd/deploy.go]: Fixed

**Comment by smarterclayton** [comment]
LGTM [merge]

**Comment by ironcladlou** [comment]
cc @thoraxe @sdodson: this is a critical fix made in the past week that you guys might need to be aware of.

**Pull request merged**